.github/workflows/docs-dead-links.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         WORKFLOW_NAME: ${{ github.workflow }}
-        WORKFLOW_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}/job/${{ github.job }}
+        WORKFLOW_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
Should this have the job id injected instead of removed?

Reply:
Github doesn't expose the info we need here so I'm changing it to just point to the workflow. You'll need to click on the failing job to see the results.